<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FunctionsWidget</class>
 <widget class="QDockWidget" name="FunctionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>289</width>
    <height>359</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="windowTitle">
   <string notr="true">Functions</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>1</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="CutterTreeView" name="functionsTreeView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">
CutterTreeView::item
{
   padding-top: 1px;
   padding-bottom: 1px;
}
       </string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="indentation">
       <number>8</number>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QuickFilterView" name="quickFilterView" native="true"/>
    </item>
   </layout>
  </widget>
  <action name="actionDisasAdd_comment">
   <property name="text">
    <string>Add comment</string>
   </property>
  </action>
  <action name="actionFunctionsRename">
   <property name="text">
    <string>Rename</string>
   </property>
  </action>
  <action name="actionFunctionsUndefine">
   <property name="text">
    <string>Undefine</string>
   </property>
  </action>
  <action name="action_References">
   <property name="text">
    <string>X-Refs</string>
   </property>
   <property name="toolTip">
    <string>Cross references</string>
   </property>
  </action>
  <action name="actionHorizontal">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Horizontal</string>
   </property>
  </action>
  <action name="actionVertical">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Vertical</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CutterTreeView</class>
   <extends>QTreeView</extends>
   <header>widgets/CutterTreeView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QuickFilterView</class>
   <extends>QWidget</extends>
   <header>widgets/QuickFilterView.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
